### PR TITLE
Update files and instructions for running in production environment

### PR DESCRIPTION
**Pull request by Larox**
This PR includes:

- Add `DEBUG` variable to `.env.example` file
- Update `Readme.md` instructions for `DEBUG` variable

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -1,3 +1,4 @@
+DEBUG=True
```
[the remark this answers is not in the document]
Comment (Larox) [reply .env.example:1]: That's right updating it.

**Review by jefflinwood — CHANGES_REQUESTED** [review]
Hi, This needs one minor wording change and then it can be merged.

File: README.md [hunk 2]
```diff
@@ -30,7 +30,8 @@ Begin by creating a configuration file for your application:
 cp .env.example .env
 ```
 
-Edit `.env` with the three configuration parameters we gathered from above.
+Edit `.env` with the three configuration parameters we gathered from above. The application runs
+by default in `production` environment. Feel free to update `DEBUG` to False if needed.
```
[the remark this answers is not in the document]
Comment (Larox) [reply README.md]: Updated!

**Comment by jefflinwood** [comment]
Thanks!

**Pull request merged**